docs/deploy.md

```diff
@@ -76,3 +75,60 @@ Finally, add a `Procfile` to your project with the following:
 web: node bin/rendertron
 ```
 
+# Deploying Rendertron in a docker container
+
+Based on Puppeteer instructions we can create a docker image that bundles a headless chrome and rendertron. We can start from node 14 base image.
+
+For more information about chrome installation please see the pupeteer page: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker
+
+Before creating the docker file you must create the config.json used by rendertron.
```

Comment:
Is that a hard requirement for running Rendertron from Docker or would it work without creating a custom config.json (Rendertron works out of the box without that file)

Reply:
Yes it is not a requirement to have this file. I will rephrase this part a bit to indicate that it is an optional step